### PR TITLE
Reminder mailer

### PR DESCRIPTION
**Pull request by hihihaha**

### CODE REVIEW

File: spec/mailers/reminder_mailer_spec.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+require "rails_helper"
+
+RSpec.describe ReminderMailer, type: :mailer do
+  let!(:user) { create(:user) }
+  let!(:thing) { create(:thing, title: "My thing", user: user) }
```
Comment (Fahrenhei7) [inline spec/mailers/reminder_mailer_spec.rb]: use single quotes instead of double

File: app/views/reminder_mailer/reminder.html.slim [hunk 2]
```diff
@@ -0,0 +1,6 @@
+| Вы не забыли, что вы хотели в ближайшие 30 дней позаниматься
```
Comment (Fahrenhei7) [inline app/views/reminder_mailer/reminder.html.slim]: How about the English language in Interface, @Freika?

File: app/mailers/reminder_mailer.rb [hunk 3]
```diff
@@ -0,0 +1,8 @@
+class ReminderMailer < ApplicationMailer
+  def reminder(user)
+    @user = user
+    @occupations = @user.occupations.includes(:thing).where(status: :in_progress)
+    mail to: @user.email,
+      subject: 'Ваши занятия'
```
Comment (Freika) [inline app/mailers/reminder_mailer.rb]: Why subject is on a new line?

File: app/views/reminder_mailer/reminder.html.slim (hunk 2 above)
Comment (Freika) [inline app/views/reminder_mailer/reminder.html.slim]: Totally agree. Use only english for now.

File: app/views/reminder_mailer/monthly_reminder.html.slim [hunk 4]
```diff
@@ -0,0 +1,4 @@
+| Hi! You have been working on #{@occupations.size > 1 ? "these things" : "this thing"} for a month. It would be great if you could share your thoughts!
```
Comment (Freika) [inline app/views/reminder_mailer/monthly_reminder.html.slim]: Use helper for pluralization

File: app/helpers/mailer_helper.rb [hunk 5]
```diff
@@ -0,0 +1,5 @@
+module MailerHelper
+  def this_thing(n)
+    n.size > 1 ? "these things" : "this thing"
```
Comment (Freika) [inline app/helpers/mailer_helper.rb]: `n.many? ? ... : ...`

**Pull request merged**